Dockerfile

```diff
@@ -1,11 +1,21 @@
-FROM node:12
+# GENERIC DOCKER SETUP
+FROM node:16
```

Comment:
Fwiw, I don't think most engineering teams have adopted Node 16 because it isn't yet supported in Lambda. Our Jenkins builder agents are configurable to use Node 12 or Node 14 currently. Not sure if this matters for your use cases, but thought I would mention.